.golangci.yml

```diff
@@ -16,7 +16,6 @@ linters:
     - dogsled
     # Check for two durations multiplied together.
     - durationcheck
-    - exportloopref
```

Comment:
removing this linter because its no longer applicable in go1.22+ https://go.dev/blog/loopvar-preview